Hcaptcha.js

```diff
@@ -35,76 +36,79 @@ const Hcaptcha = ({
   url,
   languageCode,
   cancelButtonText = 'Cancel',
-  showLoading = false,
-  loadingIndicatorColor = null,
+  showLoading,
+  loadingIndicatorColor,
   backgroundColor,
+  theme,
 }) => {
-  const generateTheWebViewContent = useMemo(
-    () =>
-      `<!DOCTYPE html>
-			<html>
-			<head> 
-				<meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0">
-				<meta http-equiv="X-UA-Compatible" content="ie=edge"> 
-				<script src="https://hcaptcha.com/1/api.js?render=explicit&onload=onloadCallback&hl=${
-          languageCode || 'en'
-        }&host=${
-        siteKey || 'missing-sitekey'
-      }.react-native.hcaptcha.com" async defer></script> 
-				<script type="text/javascript"> 
-				var onloadCallback = function() {
-			        try {
-			          console.log("challenge onload starting");
-			          hcaptcha.render("submit", {
-			            sitekey: "${siteKey || ''}",
-			            size: "invisible",
-			            "callback": onDataCallback,
-			            "close-callback": onCancel,
-			            "open-callback": onOpen,
-			            "expired-callback": onDataExpiredCallback,
-			            "chalexpired-callback": onChalExpiredCallback,
-			            "error-callback": onDataErrorCallback
-			          });
-			          // have loaded by this point; render is sync.
-			          console.log("challenge render complete");
-			        } catch (e) {
-			          console.log("challenge failed to render");
-					  window.ReactNativeWebView.postMessage("error");
-			        }
+  const customTheme = typeof theme === 'object';
+  if (!customTheme) {
+    theme = `"${theme}"`;
+  }
 
-			        try {
-			          console.log("showing challenge");
-			          hcaptcha.execute();
-			        } catch (e) {
-			          console.log("failed to show challenge");
-					  window.ReactNativeWebView.postMessage("error");
-			        }
+  const generateTheWebViewContent = useMemo(
+    () => 
+     `<!DOCTYPE html>
+      <html>
+      <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <script src="https://hcaptcha.com/1/api.js?render=explicit&onload=onloadCallback&hl=${languageCode || 'en'}&host=${siteKey || 'missing-sitekey'}.react-native.hcaptcha.com&custom=${customTheme}" async defer></script>
+        <script type="text/javascript">
+          var onloadCallback = function() {
+            try {
+              console.log("challenge onload starting");
+              hcaptcha.render("submit", {
+                sitekey: "${siteKey || ''}",
+                size: "invisible",
+                theme: ${theme},
```

Comment:
Seems cleaner to build this object separately, since we don't want to include unnecessary fields.